s3transfer/manager.py

```diff
@@ -27,7 +28,9 @@ def __init__(self,
                  multipart_threshold=8 * MB,
                  max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0):
+                 max_queue_size=0,
```

Comment:
Ah sorry I misread the default.  Have you experimented with various buffer sizes?  The CLI uses 1MB for its chunk sizes. Is there a reason we should deviate from that?